rest-service/manager_rest/dsl_functions.py

```diff
@@ -154,10 +154,28 @@ def get_secret_from_provider(secret):
                 provider.connection_parameters,
             ),
         )
+        url = connection_parameters.get('url')
+        token = connection_parameters.get('token')
+        default_path = connection_parameters.get('path')
+        secret_path = None
+
+        if secret.provider_options:
+            provider_options = json.loads(
+                decrypt(
+                    secret.provider_options,
+                ),
+            )
+            secret_path = provider_options.get('path')
+
+        path = f'{default_path}'
```

Comment:
if default path is not provided, then this will become `path = 'None'`, and that's not really what we want...